KarhooUISDK/Screens/Rides/RideDetails/RideDetailsViewController.swift

```diff
@@ -11,18 +11,33 @@ import KarhooSDK
 
 final class RideDetailsViewController: UIViewController, RideDetailsView {
 
-    private var scrollView: UIScrollView!
-    private var rideDetailsView: RideDetailsViewContainer!
-
-    private var loadingView: LoadingView!
+    // MARK: - Properties
     
     private let presenter: RideDetailsPresenter
     private var trip: TripInfo?
     private var actionButtonsPresenter: RideDetailsStackButtonPresenter?
     private var cancelRideBehaviour: CancelRideBehaviourProtocol?
     private var tripMetaDataPresenter: TripMetaDataPresenter?
     private let loadingViewAlpha: CGFloat = 0.85
-    
+
+    // MARK: Views
+
+    private lazy var scrollView = UIScrollView().then { scrollView in
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        scrollView.accessibilityIdentifier = "scroll_view"
+        scrollView.backgroundColor = .white
+    }
+    private lazy var rideDetailsView = RideDetailsViewContainer()
+    private lazy var trackDriverButton = MainActionButton().then {
```

Comment:
This new button should be visible only when the driver is trackable, i.e. the ride is in progress. Once the ride is finished or cancelled, the button should be hidden.